stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -108,6 +114,9 @@ extension ${Self}: CustomStringConvertible {
 ${Availability(bits)}
 extension ${Self}: CustomDebugStringConvertible {
   /// A textual representation of the value, suitable for debugging.
+  ///
+  /// This property has the same value as the `description` propyrty, except
```

Comment:
*property

Reply:
That's an outdated diff — this spelling error was already fixed in commit b92a5bd290a3f0cbbe80028aa9bb6bbe67973209.